test/src/main/java/org/apache/accumulo/test/ImportExportIT.java

```diff
@@ -294,6 +295,7 @@ public void testExportImportOffline() throws Exception {
 
       // verify the new table is offline
       assertFalse(client.tableOperations().isOnline(destTable), "Table should have been offline.");
+      assertEquals(getServerContext().getTableState(TableId.of(tableId)), TableState.OFFLINE);
```

Comment:
Does the following sound correct?  Want to make sure I understand these changes.

 * Without these changes the table would be in the new state, so `client.tableOperations().isOnline(destTable)` would have returned false.  
 * The call to `client.tableOperations().online(destTable, true)` just kinda fix things for the rest of the test.

If that is true, it makes me wonder if the call `client.tableOperations().online(destTable, true)` should be a bit more strict about what table states it will transition.

Reply:
Yea I think you are understanding things correctly.

I think it could make sense to be more strict about transition states but could that cause issues if a similar situation arises where a table is stuck in a state and we want to manually transition it but if the tableOp does not allow for it, then it would be really stuck without being able to manually transition. For example if for some reason a table is stuck in NEW and we want to online it but that is not an allowed transition, then it would remain stuck in that state.

Reply:
Yeah it can fix things, but it can also hide bugs in other parts of the code.  If it were more strict maybe this bug would have been found sooner.  I think it would be good to make it more strict and leave fixing buggy situations to something else like zookeeper surgery or a specialized tool if the need arises frequently enough.